movement/napari/loader_widgets.py

```diff
@@ -164,26 +165,29 @@ def _add_points_layer(self):
         # Find rows in data array that do not contain NaN values
         bool_not_nan = ~np.any(np.isnan(self.data), axis=1)
 
-        # Style properties for the napari Points layer
-        points_style = PointsStyle(
+        # Define style for points layer
+        props_and_style = PointsStyle(
             name=f"data: {self.file_name}",
             properties=self.props.iloc[bool_not_nan, :],
         )
 
-        # Color the points by individual if there are multiple individuals
-        # Otherwise, color by keypoint
+        # Set markers' text
+        text_prop = "keypoint" if "keypoint" in self.props else "individual"
```

Comment:
perhaps the text_prop should also be "individual" if "keypoint" is in props but there is only 1 unique value (i.e. centroid tracking).

Viewing the AEON data gave me this idea. If you only have 1 keypoint/individual, it's not very useful to have "centroid" written all over the place, it would be more useful to have the individual names.

Don't feel particularly strong about this, do as you wish.

Reply:
Agreed, that was one of the aims of the PR basically as right now for multi-animal datasets, the display text also shows for all keypoints the individual name.

But I had missed this case, thanks for pointing it out. Will implement it and add a test!